basculeLogging.go

```diff
@@ -45,13 +47,17 @@ func setLogger(logger log.Logger, lf ...LoggerFunc) func(delegate http.Handler)
 					}
 				}
 				kvs, _ = candlelight.AppendTraceInfo(r.Context(), kvs)
-				ctx := r.WithContext(logging.WithLogger(r.Context(), log.With(logger, kvs...)))
+				ctx := r.WithContext(context.WithValue(r.Context(), loggerKey, log.With(logger, kvs...)))
 				delegate.ServeHTTP(w, ctx)
 			})
 	}
 }
 
 func getLogger(ctx context.Context) log.Logger {
-	logger := log.With(logging.GetLogger(ctx), "ts", log.DefaultTimestampUTC, "caller", log.DefaultCaller)
-	return logger
+	logger, ok := ctx.Value(loggerKey).(log.Logger)
```

Comment:
This should really use the contextual logger API that has now been moved to `sallust`